scipy/integrate/_quadrature.py

```diff
@@ -377,7 +377,19 @@ def _basic_simps(y, start, stop, x, dx, axis):
     return result
 
 
+
+# Note: alias kept for backwards compatibility. simps was renamed to simpsons
+# because the former is a slur in colloquial English (see gh-12924).
```

Comment:
Do we need to keep the reference in the codebase?

Reply:
I'd prefer not to have the reference.

Reply:
It seems useful to me, because in the future someone is going to come along who wants to clean up the alias and wonder why it's there to begin with.

Reply:
Do you want to remove the whole comment, or the "slur" part, or the link to the issue?

Reply:
Actually, it's a comment, so it's probably fine to keep.

Reply:
In case of a doubt, there's no need to come back to it, can just keep it as is and move on.